karma.conf.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env node */
+'use strict';
+
+module.exports = function(config) {
+  config.set({
+    files: [
+      // https://segment.com/docs/integrations/optimizely/#with-javascript
```

Comment:
I'm guessing? It's not wrapped with quotes so I'm assuming this was not meant to be actually "used"